### DISCUSSION TITLE
TVP download script

### DISCUSSION THREAD

**Discussion by subtitrariistorie** [community/community#38425] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! Please make a new script to download from TVP. Their site had been updated and the script does not work anymore. Thank you! These message is adressed to all script developers which had previously created TVP download scripts.

**Comment by ossanna16** [comment]
Hi there @subtitrariistorie and welcome to our community! Can you please explain what TVP is?

**Reply by subtitrariistorie** [reply]
Polish State Television. On Github there were a few scripts which allowed to download from their video-on-demand service. But since october most of those scripts seem to not work anymore. Maybe if their authors could update their scripts it would be great. Or the great collaborators and programmers here could make new scripts to work on TVP.

**Comment by airtower-luna** [comment, marked as answer]
In that case it's best if you post your problem in the authors' repository. Check what their documentation says about reporting problems first, but in case of doubt an issue in which you describe the problem (with a sample link & error message) should work.
